client/src/components/Admin/Jobs/index.vue

```diff
@@ -94,9 +101,78 @@ export default {
             loading: false,
             jobList: [],
             root: process.env.VUE_APP_ROOT_API,
+            createJobModal: false,
+            jobModal: false,
         }
     },
+    mounted() { this.jobDetail, this.selectedStatus },
     methods: {
+        jobDetail(val) {
+            const isEmptyObjectValuesEmpty = Object.values(val).every(value => {
+                return value === '' || value === null || value === undefined;
+            });
+            if (isEmptyObjectValuesEmpty == true) {
+                this.$bvToast.toast("Please fill all the details", {
+                    title: "Error",
+                    variant: "danger",
+                    solid: true,
+                });
+            }
+            else {
+                Object.assign(val, { adminId: localStorage.getItem("adminId") });
+                this.$axios
+                    .post(`${this.root}/job/create`, val)
+                    .then((response) => {
+                        const responseData = response.data;
+                        if (responseData.status) {
+
+                            console.log("success");
```

Comment:
toast missing, also remove console.log() wherever you have placed in code, except in catch `err`